lib/commands/account/usage.ts

```diff
@@ -0,0 +1,47 @@
+import { AccountCommandBase } from "./account-command-base";
+import { UNLIMITED } from "../../constants";
+import { createTable, stringifyWithIndentation } from "../../helpers";
+
+export class UsageCommand extends AccountCommandBase implements ICommand {
+	public allowedParameters: ICommandParameter[] = [];
+
+	public get dashedOptions() {
+		return this.$nsCloudOptionsProvider.dashedOptions;
+	}
+
+	constructor($errors: IErrors,
+		$nsCloudUserService: IUserService,
+		private $nsCloudAccountsService: IAccountsCloudService,
+		private $nsCloudOptionsProvider: ICloudOptionsProvider,
+		private $options: ICloudOptions,
+		private $logger: ILogger) {
+		super($errors, $nsCloudUserService);
+	}
+
+	public async execute(args: string[]): Promise<void> {
+		const usage = await this.$nsCloudAccountsService.getUsageInfo(this.$options.accountId);
+		let output: string;
+
+		if (this.$options.json) {
+			output = stringifyWithIndentation(usage);
+		} else {
+			const table = createTable(["Feature", "Remaining usage"], usage.map(u => {
```

Comment:
Since the command is named `usage` I'd expect the full usage to be listed - builds done as well as builds remaining.
So I'd suggest `Feature`, `Builds performed` and `Builds remaining` for example.

Note that this would impact the `json` output you provide also.